configure/CONFIG_SITE

```diff
@@ -16,8 +16,10 @@ CROSS_COMPILER_TARGET_ARCHS += linux-arm_elhf
 
 # This can either be an explicit path to the directory containing the
 # epicsdbbuilder module or a version number understood by pkg_resources.require
-EPICSDBBUILDER ?= 1.3
+EPICSDBBUILDER ?= /dls_sw/prod/python3/RHEL7-x86_64/epicsdbbuilder/1.5
 
-PYTHON ?= dls-python
+SPHINXBUILD ?= /dls_sw/prod/python3/RHEL7-x86_64/sphinx/2.3.1/prefix/bin/sphinx-build
+
+PYTHON ?= python3
```

Comment:
Need to make this same change to `CONFIG_SITE` in `examples/basic_ioc` and `examples/example_ioc`.

Would be good to include builds of these two in the CI workflow.

Reply:
Changes made, and examples added to CI build